circle2/deploy-java-service.sh

```diff
@@ -15,7 +17,13 @@ export CURRENT_VERSION=$(mvn -q -Dexec.executable="echo" -Dexec.args='${project.
 TIMESTAMP=$(date +"%Y%m%d-%H%M%S")
 export SPECIFIC_BRANCH=${TIMESTAMP}-${CIRCLE_BRANCH}-${CIRCLE_SHA1:0:7}
 
-do-ecs-deployment.sh
+if [[ -n "${ECS_PROD_CLUSTER}" && -n "${ECS_PREPROD_CLUSTER}" ]]; then
+	do-ecs-deployment.sh
+fi
+
+if [[ -n "${EKS_ENVIRONMENT}" && -n "${KUBERNETES_APPLICATIONS}" ]]; then
```

Comment:
1- We won't always have `$KUBERNETES_APPLICATIONS`... This env var is kind of a hack to be able to deploy more than one service based on one repo (like we do with `womply-platform-service` from which we deploy the `-sandbox` and `-real` services.
2- I find `$EKS_ENVIRONMENT` redundant given that CircleCI is already aware of the environment. Also, if somehow the ECS deployment and EKS deployment target different environments, we would be in trouble.

Reply:
@Pepelo Give that the `deploy-java-service.sh` is a stop gap solution for the interim before we switch over the logic to `womply-scripts` for handling deployments to EKS.
- Do you think its worth investing more time to try and build generic logic here ?
- The assumption behind this implementation is that any service which gets migrated to EKS, we will need to update the circleci config file to replace the ECS environment variables with EKS specific environment variables

Reply:
Good questions:
1- I think it is worth the time given that calling the `aws` commands a `grep`ing for the service should not be too hard.
2- My assumption was that the CircleCI configs for WCJ service would stay untouched (making it easier to migrate WCJ services en masse).

Reply:
@Pepelo We can discuss more on this during the parking lot.
1- The `aws` cli might be of help incase of looking at ECS but for EKS it would just give us cluster level details and not application level details. We may have to configure and rely on `kubectl` to get application level insights
2- Currently even the `wcj` services have the mentioned environment variables defined at the circleci config level to trigger deployments on `ECS`